### PR TITLE
GetById endpoint

### PR DESCRIPTION
**Pull request by GivenTeacher5**
## Describe Your Changes
Added endpoint to call a ruleset with its rules and conditions by Id
Added tests to ensure that it is functional

## Type of Change
Please delete options that are not relevant.

- [x] New feature (change that adds functionality)

## Checklist (DOD)
- [x] The code is written and passes all Acceptance criteria's
- [x] Any Zombie code is removed
- [x] Code has proper comment documentation
- [x] The code is unit tested with happy case and passes
- [x] Code is refactored for optimization
- [ ] The code has been peer-reviewed by one or more team members who did not work on it.
- [x] The code has no merge conflicts

### CODE REVIEW

File: front-end/src/styles.css
Comment (Scott-Code-19) [inline front-end/src/styles.css:1]: =

File: front-end/src/styles.css [hunk 1]
```diff
@@ -14,4 +14,5 @@ app-root {
     --bs-btn-border-color: #7749F8;
     --bs-btn-hover-bg: #653bd7;
     --bs-btn-hover-border-color: #7749F8;
+    --bs-btn-active-bg: #7247e8;
```
Comment (Scott-Code-19) [inline front-end/src/styles.css:17]: Good this is added

File: src/main/java/com/accesspoint/rulesengine/service/RulesetService.java [hunk 2]
```diff
@@ -8,7 +8,10 @@
 import com.accesspoint.rulesengine.repository.ConditionRepository;
 import com.accesspoint.rulesengine.repository.RuleRepository;
 import com.accesspoint.rulesengine.repository.RulesetRepository;
+
+import org.apache.tomcat.util.digester.Rules;
```
Comment (Scott-Code-19) [inline src/main/java/com/accesspoint/rulesengine/service/RulesetService.java:12]: Unused import statement?
Needs to be used?
If not remove

**Review by Scott-Code-19 — CHANGES_REQUESTED** [review]
- [x] Resolve comment on RulesetService.
- [ ] Add documentation to Postman Page
- [x] Add Tests on Postman Page

Otherwise seems good to me!

File: src/main/java/com/accesspoint/rulesengine/service/RulesetService.java (hunk 2 above)
Comment (GivenTeacher5) [reply src/main/java/com/accesspoint/rulesengine/service/RulesetService.java:12]: word grah

**Review by Scott-Code-19 — CHANGES_REQUESTED** [review]
Update JSON format example on Postman Documentation to include filled-out fields instead of empty.

**Review by Scott-Code-19 — APPROVED** [review]
All changes were implemented.
Looks good to me!

**Pull request merged**